hasher-matcher-actioner/webapp/src/components/settings/ActionRulesTableRow.tsx

```diff
@@ -190,6 +184,13 @@ export default function ActionRulesTableRow({
             }}>
             <IonIcon icon={checkmark} size="large" color="white" />
           </Button>{' '}
+          <Button
+            variant="secondary"
+            className="mb-2 table-action-button"
+            onClick={() => setShowDeleteActionRuleConfirmation(true)}
+            disabled={!editing}>
+            <IonIcon icon={trashBin} size="large" className="white" />
+          </Button>{' '}
           <Button
```

Comment:
nit: I think it is slightly more intuitive to have delete (bin) below cancel (X). Thoughts @schatten ?

Reply:
Would agree. @obokaiwele can you change the order?

'Save', 'Cancel', some spacing, even an `<hr>` and _then_ 'Delete'.

Reply:
@BarrettOlson, @schatten I'll reorder the buttons and add the `<hr>`